glib/src/value.rs

```diff
@@ -336,6 +336,12 @@ impl Uninitialized for Value {
     }
 }
 
+impl StaticType for Value {
+    fn static_type() -> Type {
+        Type::value()
```

Comment:
```suggestion
        unsafe { from_glib(gobject_ffi::g_value_get_type()) }
```

Why not just call the function here directly? That seems easier :)

Reply:
Because `Type`s representation is private and I couldn't find a constructor that takes a `GType`.

Reply:
Ah. `from_glib` is what you're looking for